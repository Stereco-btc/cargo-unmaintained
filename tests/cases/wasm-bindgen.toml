url = "https://github.com/rustwasm/wasm-bindgen"
rev = "0.2.87"
