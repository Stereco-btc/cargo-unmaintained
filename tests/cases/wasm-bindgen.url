https://github.com/rustwasm/wasm-bindgen
