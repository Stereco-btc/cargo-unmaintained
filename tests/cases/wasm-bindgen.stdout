example-tests (no repository)
    mozprofile (requirement: ^0.8.0, version used: 0.8.0, latest: 0.9.1)
    mozrunner (requirement: ^0.14.0, version used: 0.14.0, latest: 0.15.1)
    tokio-tungstenite (requirement: ^0.17.2, version used: 0.17.2, latest: 0.20.1)
    tower-http (requirement: ^0.3.4, version used: 0.3.5, latest: 0.4.4)
mozrunner (no repository)
    mozprofile (requirement: ^0.8, version used: 0.8.0, latest: 0.9.1)
    plist (requirement: ^0.5, version used: 0.5.5, latest: 1.5.0)
    dirs (requirement: ^2, version used: 2.0.2, latest: 5.0.1)
    winreg (requirement: ^0.5, version used: 0.5.1, latest: 0.51.0)
raytrace-parallel (no repository)
    serde-wasm-bindgen (requirement: ^0.4.3, version used: 0.4.5, latest: 0.6.0)
raytracer (no repository)
    image (requirement: ^0.12.3, version used: 0.12.4, latest: 0.24.7)
rust-webassembly-weather-reports (no repository)
    gloo (requirement: ^0.2.1, version used: 0.2.1, latest: 0.10.0)
    reqwest (requirement: ^0.10.6, version used: 0.10.10, latest: 0.11.22)
todomvc (no repository)
    askama (requirement: ^0.10.0, version used: 0.10.5, latest: 0.12.1)
wasm-audio-worklet (no repository)
    console_log (requirement: ^0.2.0, version used: 0.2.2, latest: 1.0.0)
weedle (https://github.com/rustwasm/weedle updated [..] days ago)
    nom (requirement: ^5.0.0, version used: 5.1.3, latest: 7.1.3)
lexical-core (https://github.com/Alexhuszagh/rust-lexical updated [..] days ago)
    arrayvec (requirement: ^0.5, version used: 0.5.2, latest: 0.7.4)
    bitflags (requirement: ^1.2, version used: 1.3.2, latest: 2.4.0)
proc-macro-error (https://gitlab.com/CreepySkeleton/proc-macro-error updated [..] days ago)
    syn (requirement: ^1, version used: 1.0.109, latest: 2.0.38)
kernel32-sys (https://github.com/retep998/winapi-rs updated [..] days ago)
    winapi (requirement: ^0.2.5, version used: 0.2.8, latest: 0.3.9)
ws2_32-sys (https://github.com/retep998/winapi-rs updated [..] days ago)
    winapi (requirement: ^0.2.5, version used: 0.2.8, latest: 0.3.9)
adler32 (https://github.com/remram44/adler32-rs updated [..] days ago)
    humansize (requirement: ^1.1, version used: 1.1.1, latest: 2.1.3)
atty (https://github.com/softprops/atty updated [..] days ago)
    hermit-abi (requirement: ^0.1.6, version used: 0.1.19, latest: 0.3.3)
hyper-tls (https://github.com/hyperium/hyper-tls updated [..] days ago)
    bytes (requirement: ^0.5, version used: 0.5.6, latest: 1.5.0)
    hyper (requirement: ^0.13, version used: 0.13.10, latest: 0.14.27)
    tokio (requirement: ^0.2, version used: 0.2.25, latest: 1.33.0)
multipart (http://github.com/abonander/multipart updated [..] days ago)
    quick-error (requirement: ^1.2, version used: 1.2.3, latest: 2.0.1)
    twoway (requirement: ^0.1, version used: 0.1.8, latest: 0.2.2)
fuchsia-zircon (https://fuchsia.googlesource.com/garnet updated [..] days ago)
    bitflags (requirement: ^1.0.0, version used: 1.3.2, latest: 2.4.0)
enum_primitive (https://github.com/andersk/enum_primitive-rs.git updated [..] days ago)
    num-traits (requirement: ^0.1.32, version used: 0.1.43, latest: 0.2.17)
